src/test/java/com/odysseusinc/arachne/portal/TestContainersInitializer.java

```diff
@@ -0,0 +1,84 @@
+package com.odysseusinc.arachne.portal;
+
+import static org.testcontainers.containers.PostgreSQLContainer.IMAGE;
+
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import java.io.IOException;
+import java.util.stream.Stream;
+import org.springframework.context.ApplicationContextInitializer;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.core.env.MapPropertySource;
+import org.testcontainers.containers.BindMode;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.containers.SolrContainer;
+import org.testcontainers.lifecycle.Startables;
+import org.testcontainers.shaded.com.google.common.collect.ImmutableMap;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
+
+public class TestContainersInitializer implements ApplicationContextInitializer<ConfigurableApplicationContext> {
+    private final static String CONTAINER_CONFIG_PATH = "/opt/solr/portal-configs";
+
+    private static final DockerImageName POSTGRES_IMAGE = DockerImageName.parse(IMAGE).withTag("9.6.12");
+    static PostgreSQLContainer<?> postgres = new PostgreSQLContainer<>(POSTGRES_IMAGE);
```

Comment:
postgre or postgreSql, please

Reply:
The image is called exactly "postgres" in docker registry, as well as binary if you do a local install.

Reply:
You are right